modules/local/collect_stats.nf

```diff
@@ -0,0 +1,110 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process COLLECT_STATS {
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::r-tidyverse=1.3.1 conda-forge::r-data.table=1.14.0 conda-forge::r-dtplyr=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-508c9bc5e929a77a9708902b1deca248c0c84689:0bb5bee2557136d28549f41d3faa08485e967aa1-0' :
+        'quay.io/biocontainers/mulled-v2-508c9bc5e929a77a9708902b1deca248c0c84689:0bb5bee2557136d28549f41d3faa08485e967aa1-0' } "
+
+    input:
+    
```

Comment:
According to github actions (EditorConfig), there's a lot of trailing white spaces/wrong left paddings:

```
    16: Trailing whitespace
	31: Trailing whitespace
	44: Wrong amount of left-padding spaces(want multiple of 4)
	45: Wrong amount of left-padding spaces(want multiple of 4)
	46: Wrong amount of left-padding spaces(want multiple of 4)
	47: Wrong amount of left-padding spaces(want multiple of 4)
	48: Wrong amount of left-padding spaces(want multiple of 4)
	49: Wrong amount of left-padding spaces(want multiple of 4)
	50: Trailing whitespace
	50: Wrong amount of left-padding spaces(want multiple of 4)
	51: Wrong amount of left-padding spaces(want multiple of 4)
	52: Wrong amount of left-padding spaces(want multiple of 4)
	53: Wrong amount of left-padding spaces(want multiple of 4)
	54: Wrong amount of left-padding spaces(want multiple of 4)
	77: Trailing whitespace
	89: Trailing whitespace
	94: Trailing whitespace
```

Reply:
I should have fixed them!